src/twisted/python/test/test_deprecate.py

```diff
@@ -1067,3 +1068,56 @@ def func(a=3, b=4):
             return a + b
 
         self.assertRaises(TypeError, func, a=3, b=4)
+
+
+    def test_invalidParameterType(self):
+        """
+        Create a fake signature with an invalid parameter
+        type to test error handling.
```

Comment:
Okay.  Put that information into the code so it's available to future maintainers. :)